importers/flatfile.py

```diff
@@ -0,0 +1,75 @@
+
+import logging
+
+import click
+import csv
+import sys
+
+from datetime import datetime
+from models.version import Version
+from models.issue import Issue
+from utils.savemodel import save_model
+
+
+class FlatFileImporter:
+    """
+    Import CSV file data to the database
+    """
+
+    def __init__(self, session, filename:str, target_table:str, overwrite:bool) -> None:
+        self.session = session
+        self.filename = filename
+        self.target_table = target_table
+        self.overwrite = overwrite
+
+    def import_from_csv(self) -> None:
+        """
+        Import CSV to the database
+        """
+        logging.info('import_from_csv')
+        if self.target_table and str(self.target_table).lower() in ["alias", "author", "commit", "file", "issue", 
+                                                      "metric", "model", "ownership", "project", "version"]:
+            # Overwrite option
+            if self.overwrite:
+                self.session.execute('DELETE FROM {target_table}'.format(target_table=str(self.target_table).capitalize()))
+                click.echo('Overwrite ' + str(self.target_table).capitalize() + ' table')
+
+            if self.filename:
+                # Read CSV file
```

Comment:
in a first loop you should validate all the fields one by one. Are they mandatory ? Is the type correct ? And not wait for an obscure exception to be thrown back at the face of the user.